wkcuber/downsampling.py

```diff
@@ -148,15 +148,15 @@ def downsample(
     logging.debug(
         "Found source cubes: count={} size={} min={} max={}".format(
             len(source_cube_addresses),
-            (cube_edge_len,) * 3,
+            (buffer_edge_len,) * 3,
```

Comment:
If no explicit `buffer_edge_len` is specified, the value is `None` at this point. This leads to debug messages like: `DEBUG Found source cubes: count=1 size=(None, None, None) min=(0, 0, 0) max=(0, 0, 0)`. 
How should we handle this?

Reply:
I suggest to extract https://github.com/scalableminds/webknossos-cuber/pull/141/files#diff-363a076e73484d6f38615890702e0506R224-R226 into a `determine_buffer_edge_len` method and then open the source dataset before logging these values. Then, you can read the file_len and print the buffer_edge_len afterwards properly.